packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -860,6 +863,43 @@ describe('src/cy/commands/querying', () => {
         cy.get('div:first').should('not.be.visible')
       })
 
+      it('fails get command when element is not found', function (done) {
```

Comment:
todo: revisit. I must have changed something with verifying upcoming assertions that this isn't progating the error in the UI.

| Before | After |
| -- | -- |
| <img width="487" alt="Screen Shot 2022-11-02 at 5 16 49 PM" src="https://user-images.githubusercontent.com/14099737/199613376-a9cbe366-7f2b-4307-a972-4bbf765572f6.png">| |